tests/cli/test_suite.py

```diff
@@ -190,11 +190,14 @@ def test_suite_new_empty_suite_creates_empty_suite(
     citations = suite.get_citations()
     citations[0].pop("citation_date")
     assert citations[0] == {
-        "batch_kwargs": {"datasource": "mydatasource", "path": csv},
-        "batch_markers": None,
-        "batch_parameters": None,
-        "comment": "New suite added via CLI",
-    }
+        'batch_kwargs': {
+            'datasource': 'mydatasource',
+            'path': csv,
+            'reader_method': 'read_csv'
+        },
+        'batch_markers': None,
+        'batch_parameters': None,
+        'comment': 'New suite added via CLI'}
```

Comment:
I might be missing something here - I think this test uses a pandas datasource and it's not clear to me why the reader method is here.

Reply:
I assumed pandas datasource also takes reader_method since pandas_datasource.py has `guess_reader_method_from_path`